### DISCUSSION TITLE
How to remove the buildx builder?

### DISCUSSION THREAD

**Discussion by squarism** [abiosoft/colima#269] [General]
After installing and uninstalling colima through homebrew.  I see the builder listed:

```
$ docker buildx ls
NAME/NODE       DRIVER/ENDPOINT  STATUS PLATFORMS
colima          docker-container
  colima        colima           Cannot connect to the Docker daemon at unix:///Users/me/.colima/docker.sock.
                                           Is the docker daemon running?
<other builders listed>
```

I have tried:
```
$ docker buildx rm colima
$ docker buildx rm --builder colima
$ docker buildx rm -f colima # this shouldn't be significant or needed but I tried anyway
```

Yet the builder remains listed.  I had done a `colima delete` before `brew uninstall colima` but it's still listed there.  Anyone know how to remove the builder?  I've searched the filesystem too using `mdfind` and I can't find where this is living.  It's not affecting anything, just wanted to know how to remove it or maybe a cleanup step is needed in the formula.

**Comment by abiosoft** [comment]
buildx detects the docker context and add its dynamically. 

I suspect you still have colima in the list of docker contexts.

```
docker context ls
```

If yes, simply remove the context.

```
docker context rm colima
```

**Reply by squarism** [reply]
That was it!  Thanks.
